save-orchestrator/build.gradle.kts

```diff
@@ -20,15 +22,19 @@ tasks.withType<KotlinCompile> {
 }
 
 // if required, file can be provided manually
-val saveCliVersion = getSaveCliVersion()
-@Suppress("RUN_IN_SCRIPT")
-if (!file("$buildDir/resources/main/save-$saveCliVersion-linuxX64.kexe").exists()) {
```

Comment:
What about this check now?
We will download anyway, but don't overwrite, if file already exists? Do we really need download file in this case?

Reply:
We used to query save-cli version from github on every build. Now it's saved by `getSaveCliVersion` task into a properties file, which is updated once per 10 minutes. Then, download task won't download it if it already exists, because we can assume that the file on github is the same.